data-api/fetchGuides.ts

```diff
@@ -3,7 +3,7 @@ const fg = require('fast-glob')
 var fs = require('fs')
 var path = require('path')
 
-export default async function fetchDocs() {
+export default async function fetchGuides() {
```

Comment:
I just fixed the name for this function to match its purpose (more for code searching because it is exported by `default`).